development/main.postgressql.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_ssm_parameter" "db_username" {
+  name = "/housing-finance/development/db-username"
+}
+data "aws_ssm_parameter" "db_password" {
+  name = "/housing-finance/development/db-password"
+}
+
+# HFS Postgres Master database
+module "postgres_db_master" {
+  source               = "../modules/postgres"
+  environment_name     = var.environment_name
+  db_identifier        = "housing-finance-master"
+  db_engine            = "postgres"
+  db_engine_version    = "16.1"
+  db_instance_class    = "db.t3.xlarge"
+  db_name              = "hfs-postgres-master"
+  # TODO: Replace these with a parameter group
+  db_username          = data.aws_ssm_parameter.db_username.value
+  db_password          = data.aws_ssm_parameter.db_password.value
+  vpc_id               = "vpc-0d15f152935c8716f"
+  db_allocated_storage = 1200
+  db_port              = "5432"
+  subnet_ids           = ["subnet-05ce390ba88c42bfd","subnet-0140d06fb84fdb547"]
+  storage_encrypted    = true
+  multi_az             = false
+  enabled_cloudwatch_logs_exports = ["postgresql"]
+  maintenance_window   = "tue:01:00-tue:03:00"
```

Comment:
I don't think picking Tuesday will work out well.

They said the maintenance windows are minutes long, so the 2nd number doesn't matter much, but the start at `01:00` - this is Charges nightly run - the most intensive procedure of them all. Combining this with a working day risks us doing procedure re-runs on Wednesday morning if there're any issues.

Would say Weekend is probably better (like Saturday)... so that if the charges process fails due to whatever maintenance, there's an automatic Sunday re-run to fix the balances state before the Monday letters start getting sent.

Reply:
Good point, changed to sat
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster